plugin-cgroupv2/src/k8s/utils.rs

```diff
@@ -17,27 +17,21 @@ use std::{
 use super::token::Token;
 use crate::cgroupv2::CgroupV2Metric;
 
-/// # Structure
-///
-/// Public structure `CgroupV2MetricFile` creating a log file containing cgroupv2 data about :
-/// - cpu usage
-/// - Memory usage
-///
 #[derive(Debug)]
 pub struct CgroupV2MetricFile {
-    /// Name of the pod.
+    /// Name of the pod
```

Comment:
Actually, I liked the dot :)
It matches the Rust std style. [Example](https://doc.rust-lang.org/src/core/option.rs.html#571)